.gitignore

```diff
@@ -0,0 +1,3 @@
+env
+.env 
+__pycache__/
```

Comment:
@akhfzz for gitignore template, you can see in the Github offical repository right here: https://github.com/github/gitignore

In there, you can see various programming tech .gitignore template.